apps/vanilla/src/scripts/renderFilterByType.ts

```diff
@@ -0,0 +1,34 @@
+import { Type } from '@js-camp/core/enum';
+import { PaginationOptions } from '@js-camp/core/models/paginationOptions';
+import { setDefaultSelected } from '@js-camp/core/utils';
+
+import { FILTER_TYPE_OPTIONS } from '../constants';
+import { KEY_TYPE } from '../constants/key';
+import { getValueFromLocalStorage, setValueToLocalStorage } from '../service/localStorage';
+
+import { renderListAndPaginationToUI } from './renderPagination';
+
+const selectType = document.querySelector<HTMLSelectElement>('.filter__item_select-type');
+
+/** Render filter by type.
+ * @param options Pagination options.
+ */
+export function renderFilterByType(options: PaginationOptions): void {
+  const typeOptionHTML = FILTER_TYPE_OPTIONS.map(item => `<option value="${item.title}">${item.title}</option>`).join('');
+  if (selectType === null || selectType === undefined) {
```

Comment:
In our company we prefer to check for undefined and null together, like so
```suggestion
  if (selectType == null) {
```